.github/workflows/snyk_static_analysis_scan.yaml

```diff
@@ -1,14 +1,13 @@
 name: Snyk Static Analysis Scan
 # This git workflow leverages Snyk actions to perform a Static Application 
-# Testing scan (SAST) on our first-party code upon Pull Requests to Master & 
+# Testing scan (SAST) on our first-party code upon Pull Requests to
 # Develop branches. We use this as a control to prevent vulnerabilities 
```

Comment:
```suggestion
# Testing scan (SAST) on our first-party code upon Pull Requests to the
# "develop" branch. We use this as a control to prevent vulnerabilities 
```